app/controllers/healthcare_messaging_controller.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+class HealthcareMessagingController < ApplicationController
+  # FIXME: when ID.me is working we need to use it here, but for now skip
+  #   and just rely on the http basic authentication.
+  skip_before_action :authenticate
+  before_action :authenticate_client
+  include ActionController::Serialization
+
+  ######################################################################################################################
+  ## MHV_CONFIG
+  ## Instantiates a new configuration used in establishing a session.
+  ######################################################################################################################
+  MHV_CONFIG = VAHealthcareMessaging::Configuration.new(
+    host: ENV['MHV_SM_HOST'],
+    app_token: ENV['MHV_SM_APP_TOKEN'],
+    enforce_ssl: Rails.env.production?
+  ).freeze
+
+  ######################################################################################################################
+  ## Pagingation constants
+  ######################################################################################################################
+  DEFAULT_PER_PAGE = 10
+  MAXIMUM_PER_PAGE = 100
+
+  protected
+
+  ######################################################################################################################
+  ## client
+  ## Establishes a session using the MHV correlation id for authentication.
+  ######################################################################################################################
+  def client
+    @client ||= VAHealthcareMessaging::Client.new(config: MHV_CONFIG, session: { user_id: correlation_id })
```

Comment:
for now just have correlation_id use: ENV['MHV_SM_USER_ID'] explicitly, until we can get ID.me and MVI to base this off of the authed user.